app/controllers/issues_controller.rb

```diff
@@ -0,0 +1,53 @@
+class IssuesController < ApplicationController
+  load_and_authorize_resource :project
+  load_and_authorize_resource through: :project
+
+  def show
+    @comment = Comment.new
+    @pagy, @comments = pagy(Comment.where(commentable: @issue))
+    p @pagy
+  end
+
+  def new
+    @valid_assignees = @project.company.users
```

Comment:
load through cancancan

Reply:
I've added a before filter for this variable. In future this variable will need to be set according to the project type so I think it would be better here.

Reply:
moved to project model